### DISCUSSION TITLE
[Art competition] 2-factor authentication

### DISCUSSION THREAD

**Discussion by aza-atabayev** [prosyslab-classroom/cs348-information-security#403] [Art competition]
![photo_2023-05-22_23-31-11](https://github.com/prosyslab-classroom/cs348-information-security/assets/65112104/4d7b89f3-ca29-49ed-bae1-459683eb1208)

## Short Description
For me,  *2-factor authentication* means "Two ancient guardians, each unique in their armor design and color, stand tall at the entrance of a radiant, pulsating portal. The first guardian is clad in a dark, ornate armor, adorned with intricate patterns and symbols, his halberd gleaming under the ethereal light of the portal. The second guardian, in stark contrast, is dressed in a bright, polished armor that shines like the sun, his halberd equally impressive. Both guardians have a keyhole embedded in the chest of their armor, each distinctly different - one is circular with an intricate pattern, while the other is a complex, angular design. They stand on lush green hills, the verdant landscape stretching out behind them under a clear, azure sky.".

## Long Description (up to 3 sentences)
2-factor authentication is an authentication method in which the user is granted access only after successfully presenting 2 pieces of evidence. For me, 2-factor authentication is 2 distinctly different guards that protect a magical portal to infinite possibilities. They will let you pass only if you insert 2 different keys in their keyholes in the chest.

**Comment by KihongHeo** [comment]
Oh. Diablo-style guardians. Nice.

I didn't know that DALL-E can handle such a long description.
